editor/multi_node_edit.cpp

```diff
@@ -137,14 +141,18 @@ void MultiNodeEdit::_get_property_list(List<PropertyInfo> *p_list) const {
 		List<PropertyInfo> plist;
 		n->get_property_list(&plist, true);
 
-		for (const PropertyInfo &F : plist) {
+		for (PropertyInfo F : plist) {
 			if (F.name == "script") {
 				continue; // Added later manually, since this is intercepted before being set (check Variant Object::get()).
+			} else if (F.name.begins_with("metadata/")) {
+				F.name = F.name.replace_first("metadata/", "Metadata/"); // Trick to not get actual metadata edited from MultiNodeEdit.
```

Comment:
If this hack is to prevent actual MultiNodeEdit metadata from being edited, can we use something more obvious such as "`_metadata`"? Uppercase vs lowercase may be difficult to distinguish.

Reply:
MultiNodeEdit actual metadata is not editable, so this is not a problem